docs/conf.py

```diff
@@ -15,8 +15,6 @@
 
 sys.path.insert(0, os.path.abspath(".."))
 
-import sphinx_rtd_theme
```

Comment:
This doesn't need to be imported (even though the docs tell you to), and it just makes mypy angry :/